Week3/homework/js-exercises/e4-shopping.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+// function addToShoppingCart(list, item) {
+
+
+//     if (list.length > 2) {
+//         list.push(item)
+//         return console.log(list)
+//     } else {
+//         list.shift()
+//         list.push(item)
+//         return console.log(list)
+//     }
+
+// }
+
+// const groceryItems=['bananas', 'milk']
+
+// addToShoppingCart(groceryItems,'mango')
+// addToShoppingCart(groceryItems,'Juice')
+// addToShoppingCart(groceryItems,'Lemon')
+
+function addToShoppingCart(list, item) {
+
+    list.push(item)
+
+    if (list.length > 3) {
+        list.shift()
+        return console.log(list)
```

Comment:
I think it's better to return the `list`. And then `console.log` it outside of the function, like this:
```
console.log(addToShoppingCart(groceryItems,'mango'));
```